models/turbine_models/custom_models/sd_inference/sd_pipeline.py

```diff
@@ -624,9 +622,11 @@ def produce_images_compiled(
             sample,
             prompt_embeds,
             text_embeds,
-            guidance_scale,
+            torch.as_tensor([guidance_scale], dtype=sample.dtype),
```

Comment:
Is this necessary?

Reply:
Yep, the compiled pipeline expects an `fp16` tensor while passing this produces an `fp64` one.